src/FixedDataTableColumnReorderHandle.js

```diff
@@ -71,22 +77,32 @@ var FixedDataTableColumnReorderHandle = createReactClass({
           'fixedDataTableCellLayout/columnReorderContainer': true,
           'fixedDataTableCellLayout/columnReorderContainer/active': false,
         })}
+        /**
+         * Stop the event from being propagated when touching the handle.
+         * This prevents the rows from moving around when we drag the headers.
+         */
+        onTouchStart={e => e.stopPropagation()}
```

Comment:
Some questions about 2 onTouchStart handlers.

Also don't bind these events unless touchEnabled is true

Reply:
Same as above. I am going to combine them as suggested and stopPropagation only if touch is enabled since the onMouseDown function is called onMouseDown as well.